content/altinn-studio/guides/development/signing/runtime-delegated-signing/backend-manual/setup-text-resources.en.md

```
@@ -14,7 +15,12 @@ You can name these whatever you want and connect them to `CommunicationConfig` i
 
 Example of text resources for notifications with custom texts for email, as well as receipt:
 
-```
+{{% notice warning %}}
+Please note that the text resources related to the [correspondence](#correspondence) contains an `$instanceUrl$` placeholder.
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)